crates/sui-config/src/verifier_signing_config.rs

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use move_vm_config::verifier::MeterConfig;
+use serde::{Deserialize, Serialize};
+
+// Default values for verifier signing config.
+pub const DEFAULT_MAX_PER_FUN_METER_UNITS: usize = 2_200_000;
+pub const DEFAULT_MAX_PER_MOD_METER_UNITS: usize = 2_200_000;
+pub const DEFAULT_MAX_PER_PKG_METER_UNITS: usize = 2_200_000;
+
+pub const DEFAULT_MAX_BACK_EDGES_PER_FUNCTION: usize = 10_000;
+pub const DEFAULT_MAX_BACK_EDGES_PER_MODULE: usize = 10_000;
+
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+#[serde(rename_all = "kebab-case")]
+pub struct VerifierSigningConfig {
+    #[serde(default)]
+    max_per_fun_meter_units: Option<usize>,
+    #[serde(default)]
+    max_per_mod_meter_units: Option<usize>,
+    #[serde(default)]
+    max_per_pkg_meter_units: Option<usize>,
+
+    #[serde(default)]
```

Comment:
If these are the signing-only limits, I'm guessing the configs above are used by the signing verifier but are not unique to it -- is that right? If so, could you add a doc comment to explain the distinction? Maybe on `VerifierSigningConfig` -- it would help to explain why this type exists, distinct to `MeterConfig` and `VerifierConfig`.

Reply:
Good call out. Will add a comment!